operations/mimir-mixin-compiled/dashboards/mimir-alertmanager-resources.json

```diff
@@ -873,8 +873,8 @@
                   "timeShift": null,
                   "title": "Receive bandwidth",
                   "tooltip": {
-                     "shared": false,
-                     "sort": 0,
+                     "shared": true,
+                     "sort": 2,
```

Comment:
[question] What's the impact of this change I'm seeing on tooltips? Asking cause I don't know.

Reply:
It enable the share tooltip by default.
This changes is related to [the following commit](https://github.com/grafana/jsonnet-libs/commit/ccdd25d604cacdc8a8b303774be44d6102b71e8e)

Reply:
It enables this feature:
https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v4-1/#shared-tooltip

![shared tooltip demo](https://grafana.com/static/img/docs/v41/shared_tooltip.gif)

(I'm all in for it, but it may be worth mentioning it in the changelog)

Reply:
I just tried the shared tooltip on these dashboards in a large cluster and unfortunately they're unusable. All shared tooltips cover all panels. I would revert back this change, getting back to non shard tooltips.

Reply:
Problem is with the "per pods" panels.

Reply:
_Sorry for the late reply!_

I reviewed https://github.com/grafana/jsonnet-libs/pull/775 but I don't think the tooltip settings should be part of `dashboard()` because the tooltip is something belonging to the individual panel. Couldn't we just override `panel()` on our mixin?

Reply:
I did it in this commit:
https://github.com/grafana/mimir/pull/1802/commits/bb7b748775a55c60038efcec8effc4567893dc1b